### PR TITLE
Mj redisgraph 13679

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/13679

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -117,6 +117,22 @@ oc edit pvc <storageclassname>-search-redisgraph-0
...
+//2.2.5:13679
+
+The `redisgraph` StatefulSet and pod are not refreshed when you upgrade from 2.2.z to 2.2.5. You must delete the `redisgraph` StatefulSet manually, so that the changes are picked up. Complete the following steps to fix this issue:
+
+. Install {product-title-short} 2.2.4.
+//how does the user upgrade from the console? Do they navigate to the cluster page and select the Action icon? 
+. Upgrade to 2.2.5.
+//How would the user check the redisgraph statefulset? What page do they need to go to?
+. Log in as an administrator and check the `redisgraph` StatefulSet.
+. Notice that the StatefulSet and pod did not restart.
```
Comment (SherinV) [inline release_notes/known_issues.adoc:133]: You can get the statefulset with the following command:
`oc get statefulset search-redisgraph -n open-cluster-management`

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -117,6 +117,22 @@ oc edit pvc <storageclassname>-search-redisgraph-0
...
+//Is there an action icon with a list of actions and the user can select delete?
+. Delete the `redisgraph` StatefulSet.
```
Comment (SherinV) [inline release_notes/known_issues.adoc]: Delete the statefulset using:
`oc delete statefulset search-redisgraph -n open-cluster-management`

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -117,6 +117,22 @@ oc edit pvc <storageclassname>-search-redisgraph-0
...
+. Verifiy if the `redisgraph` pod is updated and running successfully.
```
Comment (SherinV) [inline release_notes/known_issues.adoc]: Verifiy if the `redisgraph` pod is running successfully.
`oc get pod search-redisgraph-0 -n open-cluster-management`

**Review by SherinV — COMMENTED** [review]
Added commands to get and delete the statefulset. You can add it if it helps. @dockerymick

**Review by SherinV — APPROVED** [review]

**Comment by dockerymick** [comment]
Merging this PR since it was approved

**Pull request merged**